pkg/loki/loki.go

```diff
@@ -631,6 +631,7 @@ func (t *Loki) setupModuleManager() error {
 		Compactor:                {Server, Overrides, MemberlistKV, Analytics},
 		IndexGateway:             {Server, Store, IndexGatewayRing, IndexGatewayInterceptors, Analytics},
 		BloomGateway:             {Server, BloomGatewayRing, Analytics},
+		BloomCompactorRing:       {Server},
```

Comment:
better to have this instead to be in line with the other rings. but nothing about this is running, so i can change that later
`BloomCompactorRing: {Overrides, MemberlistKV},`